pydm/tests/test_display.py

```diff
@@ -32,13 +25,20 @@ def ui_filename(self):
 
 
 def test_nonexistant_ui_file_raises(qtbot):
-    with pytest.raises(IOError):
+    try:
```

Comment:
Why'd you have to roll your own check here?

Reply:
It was a 🧠 fart... I will add it back...

Reply:
Addressed via 4f460d14